requirements.txt

```diff
@@ -8,3 +8,4 @@ aiodns
 psycopg2
 tldextract
 icmplib
+python-nmap
```

Comment:
goal of port scanning module is to provide a extremely fast overview of top 1K ports which it already performs properly, everyone will use nmap in a later stage anyways, so this is just an extra dependency not serving much purpose